JesusOmarR/react-apprenticeship-capstone4
# Feat/deliverable1

## Description
```
First derivable of the fourth capstone project
I created the main layout of the app 
Created the sliders and grids needed in the Home Page
Added styles to make a responsive app

```
## Type of Change

```
Delete options that are not relevant.
```




## How should this be tested?
```
Run npm run test to see all the test cases and verify if they have a good implentatiob
```
## What is the deliverable?
```
First derivable of the fourth capstone project 
```
## Screenshots and links
```
<img width="1526" alt="image" src="https://user-images.githubusercontent.com/97203630/152934475-ea66aa35-5c8e-4789-8928-43643de75511.png">
```

## Checklist
- [ x] I added the necessary documentation, if appropriate.
- [x ] I added tests to prove that my fix is effective or my deliverable works.
- [ x] I reviewed existing Pull Requests before submitting mine.
- [x ] I have commented my code, particularly in hard-to-understand areas
